src/index.js

```diff
@@ -1,36 +1,149 @@
-const tasks = [
-  {
-    description: 'Wash the dishes',
-    completed: false,
-    index: 1,
-  },
-  {
-    description: 'Completed TO Do list project',
-    completed: true,
-    index: 2,
-  },
-];
-
-document.addEventListener('DOMContentLoaded', () => {
-  const todoList = document.getElementById('todo-list');
-  todoList.innerHTML = '';
-  todoList.style.listStyle = 'none';
-
-  tasks.sort((a, b) => a.index - b.index);
-
-  tasks.forEach((task) => {
-    const li = document.createElement('li');
-    li.style.border = '1px solid #ddd';
-    li.style.display = 'block';
-    li.style.padding = '10px 0px 10px 50px';
-    li.style.marginLeft = '-40px';
-    li.style.marginRight = '50vw';
+import './style.css';
+
+const taskInput = document.getElementById('input');
+const addButton = document.getElementById('button');
+const todoList = document.getElementById('todo-list');
+const completeButton = document.querySelector('.complete');
+
+const savedTasks = JSON.parse(localStorage.getItem('tasks')) || [];
+
+function saveTasks() {
+  localStorage.setItem('tasks', JSON.stringify(savedTasks));
+}
+
+function removeTask(li) {
+  const index = Array.prototype.indexOf.call(todoList.children, li);
+  savedTasks.splice(index, 1);
+  saveTasks();
+
+  // Update remaining tasks' indexes
+  for (let i = index; i < todoList.children.length; i += 1) {
```

Comment:
- [ ] After modifying your project, Your items don't seem to have the index property. Indexes provide a unique identifier for each item on the list. This can be helpful if you need to perform operations on individual items, such as updating or deleting them. Kindly also ensure that the indexes are being updated as required in the project

![image](https://user-images.githubusercontent.com/98453979/232208385-c098089b-af2f-4f7f-87f0-9931deab6bf0.png)

Reply:
Every item has index property. Why are you telling me this? Did you check it properly?

Reply:
- [ ] If you look at the data saved in the local storage from the browser, your app stores only the `text` and `completed` property.  as shown below :point_down: However, it should also add the index property and set it to the value of the new array length (i.e. if a 5th task is added to the list, the index of that task should equal to 5).

### missing the index property

![Screenshot from 2023-04-15 12-56-22](https://user-images.githubusercontent.com/65233860/232225347-32a27ace-16e1-4c74-a2ba-757069269dff.png)

![Screenshot from 2023-04-15 13-03-32](https://user-images.githubusercontent.com/65233860/232225387-a7781619-86ec-403e-8685-bcebdfc6bc32.png)